modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/rouny.dm

```diff
@@ -55,6 +55,9 @@
 	addtimer(CALLBACK(src, .proc/evasion_deactivate), evasion_duration)
 	evade_active = TRUE
 	RegisterSignal(owner, COMSIG_PROJECTILE_ON_HIT, .proc/on_projectile_hit)
+	REMOVE_TRAIT(owner, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+	addtimer(CALLBACK(src, .proc/give_back_ventcrawl), (evasion_duration * 5)) //They cannot ventcrawl until 5x the duration has passed (50 seconds)
```

Comment:
What if that timer is called multiple times? Shouldn't it have some flags, so they won't stack?

Reply:
Maybe, I realized it was dumb to not just go off of the cooldown time so changed it to that, which should significantly reduce the chance this ever happens to begin with (looking at you other admins)